querydsl/index.asciidoc

```diff
@@ -0,0 +1,93 @@
+= Querydsl Tutorial
+====
+This tutorial will teach you how to integrate link:https://querydsl.com/[Querydsl] into a Spring or Quarkus project and how to build Querydsl queries.
+## Prerequisites
+* Installed devonfw IDE
+* User should have basic Java development experience
+
+## Learning goals.
+Here in this tutorial you will learn 
+* To integrate Querydsl in your project
+* To build Querydsl expressions  
+====
+
+[step]
+--
+restoreDevonfwIde(["java","mvn"])
+--
+
+====
+devonfw IDE has been installed for you.
+
+First, clone the `QueryDslTutorial` repository from GitHub. It contains an application with a simple REST service.
+[step]
+== Clone QueryDslTutorial repository
+--
+cloneRepository("", "https://github.com/EduardKrieger/QueryDslTutorial.git")
+--
+
+In the next step, integrate Querydsl into your Maven project.
+====
+
+[step]
+== Add Querydsl dependencies to Maven Project
```

Comment:
It would be helpful to have some additional text here that describes for what and why the dependencies and the annotation processor are needed